### PR TITLE
fix(ui): select, date-input, time-input keyboard navigation

### PR DESCRIPTION
**Pull request by m0ksem**
Previously it was impossible to open select, date-input, time-input with space and enter for some reason.
closes https://github.com/epicmaxco/vuestic-ui/issues/3533
closes https://github.com/epicmaxco/vuestic-ui/issues/3530

### CODE REVIEW

File: packages/ui/src/components/va-date-input/VaDateInput.vue [hunk 1]
```diff
@@ -133,7 +133,7 @@ import { VaIcon } from '../va-icon'
 const VaInputWrapperProps = extractComponentProps(VaInputWrapper, ['focused', 'maxLength', 'counterValue'])
```
Comment (asvae) [inline packages/ui/src/components/va-date-input/VaDateInput.vue:133]: image

Date input doesn't close on click outside.

Comment (asvae) [reply packages/ui/src/components/va-date-input/VaDateInput.vue:133]: Same issue for time input

image

**Review by asvae — CHANGES_REQUESTED** [review]
Checked locally - there are some issues.

**Review by pogrib0k — CHANGES_REQUESTED** [review]
I looked it up like everything is ok.
But `DateInput` allows focus outside the component, so sometimes something like this happens:
image

Also, because of this, I can open many "dropdowns" at the same time (see screenshot), and when I press **ESC** they all close at the same time.
image

**Review by ilearnjs — APPROVED** [review]

**Pull request merged**